src/node_modules/@types/node/globals.d.ts

```diff
@@ -0,0 +1,1171 @@
+// This needs to be global to avoid TS2403 in case lib.dom.d.ts is present in the same build
```

Comment:
There are many unnecesarry files in `/src/node_modules` which are generated unintentionally with installing nativescript types and which should be set in `.gitignore`

### Procedure for removing it

- delete entire directory `src/node_modules`.
- make a commit where all these files will be deleted.
- open `.gitignore` and set new line `src/node_modules`
- make another commit with `.gitignore` changes.

This way, files will be deleted and will be set inside git ignore so we'll never have this issue again.